<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>639</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans Mono</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>SerialPortDebugger</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>17</y>
     <width>761</width>
     <height>609</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QTextBrowser" name="recvTextBrowser">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>550</width>
          <height>400</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>3</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTextEdit" name="sendTextEdit">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>550</width>
          <height>150</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>3</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="sendFilePath">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>550</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
          <pointsize>9</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>DejaVu Sans Mono</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>串口选择：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>DejaVu Sans Mono</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>波特率：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>DejaVu Sans Mono</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>数据位：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="serialPortSel">
          <property name="font">
           <font>
            <family>DejaVu Sans Mono</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="dataBitSel">
          <property name="font">
           <font>
            <family>DejaVu Sans Mono</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>DejaVu Sans Mono</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>停止位：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="baudRateSel">
          <property name="font">
           <font>
            <family>DejaVu Sans Mono</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>DejaVu Sans Mono</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>校验位：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="stopBitSel">
          <property name="font">
           <font>
            <family>DejaVu Sans Mono</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="checkBitSel">
          <property name="font">
           <font>
            <family>DejaVu Sans Mono</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>196</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <widget class="QPushButton" name="fontBtn">
          <property name="text">
           <string>字体</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="backGrndBtn">
          <property name="text">
           <string>背景颜色</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="fontColorBtn">
          <property name="text">
           <string>字体颜色</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="openSerialPortBtn">
          <property name="text">
           <string>打开串口</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>196</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="hexDisplay">
          <property name="font">
           <font>
            <family>DejaVu Sans Mono</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>16进制显示</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="timeStamp">
          <property name="font">
           <font>
            <family>DejaVu Sans Mono</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>时间戳[以换行'\n'分割]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>196</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QPushButton" name="sendBtn">
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="stopSendBtn">
          <property name="text">
           <string>停止发送</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="cleanSendArea">
          <property name="text">
           <string>清空发送</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="cleanDisplayArea">
          <property name="text">
           <string>清空接收</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="hexSend">
            <property name="font">
             <font>
              <family>DejaVu Sans Mono</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>16进制发送</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="sendInterval">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>DejaVu Sans Mono</family>
              <pointsize>11</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="sendNewLine">
            <property name="font">
             <font>
              <family>DejaVu Sans Mono</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>发送新行</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="schedSend">
            <property name="font">
             <font>
              <family>DejaVu Sans Mono</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>定时发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="openFileBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>DejaVu Sans Mono</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>打开文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendFileBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>发送文件</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
